src/js/control-bar/fullscreen-toggle.js

```diff
@@ -12,6 +12,10 @@ import Component from '../component.js';
  */
 class FullscreenToggle extends Button {
 
+  constructor(player, options) {
+    super(player, options);
+    player.on('fullscreenchange', this.handleFullscreenChange);
```

Comment:
if you change this to `this.on(player, 'fullscreenchange', this.handleFullscreenChange);` it'll make the context of the handler be `this` (the fullscreen toggle) plus it'll manage the event handler better since it'll be removed when this component is disposed and not necessarily just when the player is disposed.